fragdenstaat_de/fds_blog/templatetags/blog_tags.py

```diff
@@ -12,3 +12,11 @@ def get_next_read(article):
     if article.start_publication:
         return qs.filter(start_publication__lt=article.start_publication).first()
     return qs.exclude(pk=article.pk).first()
+
+
+@register.inclusion_tag("fds_blog/blog_preview.html", takes_context=True)
+def get_blog_preview(context, amount=6):
+    amount = min(int(amount), 20)
+    articles = Article.published.all()[:amount]
```

Comment:
Would something like this work as well, as upper bounds higher than the maximum value should be handled gracefully[^1]?

```suggestion
    articles = Article.published.all()[:int(amount)]
```

[^1]: https://docs.python.org/3/tutorial/introduction.html#:~:text=However%2C%20out%20of%20range%20slice%20indexes%20are%20handled%20gracefully%20when%20used%20for%20slicing%3A